### ISSUE TITLE
[Bug]: Alpha branch themes need a revisit

### ISSUE THREAD

**Issue by Smurf-IV** [Krypton-Suite/Standard-Toolkit#883]
Please get latest alpha branch and run the following to see the problems:

![image](https://user-images.githubusercontent.com/2418812/209323099-3420d5ec-3f66-4a78-a2f1-1aa1ea3f28ca.png)

**Comment by Wagnerp** [comment]
Linked to #880. @Smurf-IV I really think that `SchemeOfficeColors` needs to be sorted out first.

**Comment by Wagnerp** [comment]
As all content text _should_ not all share the same colour value.

**Comment by Smurf-IV** [comment]
Yes:
![image](https://github.com/Krypton-Suite/Standard-Toolkit/assets/2418812/dfa6a507-6a4f-4159-8608-726374d5ae7e)

**Comment by Wagnerp** [comment]
@Smurf-IV & @giduac 

Which ones need attention the most?

**Comment by Wagnerp** [comment]
I have a feeling that all sparkle checkboxes/radio buttons are using KButton colours. Why, I don't know  yet?

**Comment by Smurf-IV** [comment]
> @Smurf-IV & @giduac
> 
> Which ones need attention the most?

You can see the Sparkle ones above, But the Office ones for the Active tab, and the ribbon do not go together either (As per Word / office examples)

**Comment by Wagnerp** [comment]
> Please get latest alpha branch and run the following to see the problems:
> 
> ![image](https://user-images.githubusercontent.com/2418812/209323099-3420d5ec-3f66-4a78-a2f1-1aa1ea3f28ca.png)

@Smurf-IV & @giduac 

Is this still the same... as...

![image](https://github.com/user-attachments/assets/21bcb49e-3002-4e96-85f1-c44474f9d8f2)

![image](https://github.com/user-attachments/assets/c02f813c-847b-4c84-a9f2-71735ea5a9e5)

![image](https://github.com/user-attachments/assets/c692a4f6-de6d-481b-a923-9e1058b3c03c)

?

Using todays dogfood build

**Comment by Smurf-IV** [comment]
Active "tab" is supposed to have the same colouring ass the ribbon background.
The other tabs that do not have focus, should be of another colour, as per Word Office examples.

**Comment by Wagnerp** [comment]
@Smurf-IV & @giduac 

What's the best way of finding out which enum value radio buttons & checkboxes use?

**Comment by PWagner1** [comment]
Hi @Smurf-IV & @giduac 

I can't seem to find where the bug is for this:

![image](https://github.com/user-attachments/assets/2bbaa54f-4018-46fd-a8a7-ba99e10f9257)

Would you mind taking a look, as I've spent hours trying to do so?

**Comment by Smurf-IV** [comment]
> Hi @Smurf-IV & @giduac
> 
> I can't seem to find where the bug is for this:


Not sure what you mean?
Run the toolkit example, and then select the "Sparkle - Blue"
Check in V85 and V90 branches

**Comment by PWagner1** [comment]
> > Hi @Smurf-IV & @giduac
> > I can't seem to find where the bug is for this:
> 
> Not sure what you mean? Run the toolkit example, and then select the "Sparkle - Blue" Check in V85 and V90 branches

Hi @Smurf-IV 

> Run the toolkit example, and then select the "Sparkle - Blue" Check in V85 and V90 branches

Yes, but the value that's supposed to control the text colour doesn't seem to reflect the changes. I have reset the palette colours back to a known working version, 4.6.0.0, but that does not seem to work?

**Comment by giduac** [comment]
Then that PR should be reverted if it doesnt work: https://github.com/Krypton-Suite/Standard-Toolkit/pull/1761

**Comment by PWagner1** [comment]
> Then that PR should be reverted if it doesnt work: #1761

Not really, nothing changed in the two versions were identical before changes were made, except for the formatting clean up. It makes me wonder if this has nothing to do with the palettes?

**Comment by giduac** [comment]
If the PR doesn't change anything then it has no purpose in the first place.
And that you discovered that it isn't working AFTER the PR was posted and merged point to the fact that is wasn't tested well enough....

To avoid further problems going back to the previous state is highly recommended. And there's no shame in doing that. Actually it is what all programmers do !!

**Comment by PWagner1** [comment]
@Smurf-IV & @giduac 

I may have a 'fix' but it may mean that the colour is harder to read on a panel.

**Comment by PWagner1** [comment]
This is starting to drive me up the wall!

**Comment by giduac** [comment]
> This is starting to drive me up the wall!

Have good long think about the earlier offer to tackle the theme problems.
And then let me know somewhere next week....

**Comment by PWagner1** [comment]
> > This is starting to drive me up the wall!
> 
> Have good long think about the earlier offer to tackle the theme problems. And then let me know somewhere next week....

It looks as if controls behave differently when placed in a container i.e. a KNavigator from other containers like a KPanel. If you try to change the colours, they work/update correctly when on a KPanel but not in a KNavigator. Not entirely sure why?

**Comment by PWagner1** [comment]
![Bug 883](https://github.com/user-attachments/assets/9832a3f6-2c25-4adb-85d2-642f47a1b266)

**Comment by PWagner1** [comment]
@Smurf-IV or @giduac 

Do you want to take a peek at this, as I'm really busy on work related projects ATM, so can't really dedicate a lot of time to Krypton, other than odd bugfixes here & there?

**Comment by Smurf-IV** [comment]
OK this was a 2 fold problem:
1st is solved by #1905

2nd is that the example code forces an override of the back style to something that does not make sense.

Due to 2 this is moving to the Demo's repo.
